pkgs/development/python-modules/pyxdg/default.nix

```diff
@@ -1,30 +1,33 @@
-{ stdenv
+{ lib
 , buildPythonPackage
-, fetchPypi
-, fetchpatch
+, fetchFromGitLab
+, nose
 }:
 
 buildPythonPackage rec {
   pname = "pyxdg";
-  version = "0.26";
+  version = "0.27";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06";
+  src =  fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "xdg";
+    repo = pname;
+    rev = "rel-${version}";
+    sha256 = "1dg826vrc7ifkk4lnf648h61cqfamaqmngkn9hgmxnf9gqmkbn0k";
   };
 
-  # error: invalid command 'test'
   doCheck = false;
 
-  patches = [
-    # see: https://gitlab.freedesktop.org/xdg/pyxdg/-/merge_requests/5
-    (fetchpatch {
-      url = "https://gitlab.freedesktop.org/xdg/pyxdg/-/commit/78405aaa34463db2c6f33ca28ae2293dd3bb1e91.patch";
-      sha256 = "17cjax546rkqv5kvwczjqjdd6vmlvcxjanz0296dlfq23j2wbx63";
-    })
-  ];
+  #Currently these tests fail (4 errors, 4 warnings)
```

Comment:
Please remove the commented out code.

Reply:
Done, the tests are still failing and I've removed references to the tests.